environment/deployments/science-platform/env/dev-cloudsql.tfvars

```diff
@@ -3,20 +3,29 @@ environment      = "dev"
 application_name = "science-platform"
 project_id       = "science-platform-dev-7696"
 
-# Butler database
-butler_db_name     = "butler-registry-dev"
-butler_require_ssl = false
-butler_database_flags = [
+# Butler Registry DP02 Database
+butler_db_name                         = "butler-registry-dp02-dev"
```

Comment:
This variable name doesn't match the place it is consumed: `butler_registry_dp02_db_name`

Reply:
Updated